modules/claims_api/config/schemas/v2/526.json

```diff
@@ -189,7 +189,7 @@
         "city": {
           "description": "City for the Veteran's new address.",
           "type": "string",
-          "pattern": "^$|^([-a-zA-Z'.' ])+$",
+          "pattern": "^$|([a-zA-Z0-9\\'-'.# ]([a-zA-Z0-9\\'-'.# ])?)+$",
           "maxLength": 1000,
```

Comment:
Should we make `"minLength": 0` and `"maxLength": 30`?

Reply:
This ticket was changed while I was working on it and I did not know that the min and max length was added for this attribute as well. TY